content/api-streamer/consumer.textile

```diff
@@ -0,0 +1,34 @@
+---
+title: Consumer
+section: api-streamer
+index: 68
+---
+
+API Streamer has two roles: a producer and consumer. A consumer is usually invited to API Streamer by a producer so the consumer can subscribe to, manage, and consume the data streams. A data stream could be anything from live transport updates to a stream of sensor data from an embedded IoT device.
+
+h2(#why-ably). Why did my data producer choose to use Ably for streaming data?
+
+Producers choose to expose their realtime APIs through Ably's API Streamer as it makes it easier for both producers and consumers to share realtime data. 
+
+Using API Streamer to consume data streams:
+
+* significantly lowers integration time and engineering cost. You can use the protocols or adapters available to simplify data ingestion.
+
+* allows interoperability for open streaming protocols from SSE, to MQTT, to STOMP.
+
+* allows APIs to be event-driven, capable of pushing data directly into queueing or data processing engines, legacy systems, or even trigger serverless functions in the cloud. For example, Apache Kafka, Amazon Kinesis, or Microsoft Azure.
+
+* offers an end-to-end solution for delivering data to systems or even end-user devices. With current APIs, consumers must process data from producers, store it, and publish it for their end users. API Streamer can enable delivery of data from source to subscriber.
+
+* does away with polling data. Pub/Sub architecture provides high-performance realtime APIs where new updates are pushed directly to you without you having to pull them.
```

Comment:
This section feels like a marketing section, yet we have landing pages.  I thought the documentation is aimed at developers, focussing on tangible (non opinionated) benefits, and how they get started with the API streamer.  I fear this documentation is some times too high level, and often marketing focussed.  Our documentation needs to give **developers** an overview of concepts / the service, provide the detail they need to understand what we offer (they are techies) and give them the info they need to get started now.